controllers/release/release_adapter.go

```diff
@@ -222,24 +222,20 @@ func (a *Adapter) EnsureSnapshotEnvironmentBindingIsCreated() (results.Operation
 // EnsureTargetContextIsSet is an operation that will ensure that the targetContext of the adapter is not nil. It will
 // set it to a context from the workspace in the ReleasePlan if one is defined or to its own context if not.
 func (a *Adapter) EnsureTargetContextIsSet() (results.OperationResult, error) {
-	if a.targetContext == nil {
-		releasePlan, err := a.getReleasePlan()
-		if err != nil {
-			patch := client.MergeFrom(a.release.DeepCopy())
-			a.release.MarkInvalid(v1alpha1.ReleaseReasonReleasePlanValidationError, err.Error())
-			return results.RequeueOnErrorOrStop(a.client.Status().Patch(a.context, a.release, patch))
-		}
-
-		if releasePlan.Spec.Target.Workspace == "" {
-			a.logger.Info("Running in a traditional kubernetes environment")
-			a.targetContext = a.context
-		} else {
-			a.logger.Info("Running in a KCP environment")
-			a.targetContext = logicalcluster.WithCluster(a.context, logicalcluster.New(releasePlan.Spec.Target.Workspace))
-		}
+	if a.targetContext != nil {
+		a.logger.Info("target context is set")
```

Comment:
We have to revisit logging in the future but I'd say that we want to skip any logging for the context. It's so annoying when going through the logs as it's always there for every single reconcile.